app/components/post_list/post/footer/footer.tsx

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useMemo} from 'react';
+import {TouchableOpacity, View} from 'react-native';
+
+import {updateThreadFollowing} from '@actions/remote/thread';
+import CompassIcon from '@components/compass_icon';
+import FormattedText from '@components/formatted_text';
+import UserAvatarsStack from '@components/user_avatars_stack';
+import {preventDoubleTap} from '@utils/tap';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+import type ThreadModel from '@typings/database/models/servers/thread';
+import type UserModel from '@typings/database/models/servers/user';
+
+type Props = {
+    participants: UserModel[];
+    serverUrl: string;
+    teamId: string;
```

Comment:
I see the teamId being queried in the Post component regardless of having CRT on or not and I wonder if it would be best to pass the postId instead to the Footer component and query the teamId in this component instead. in fact I do believe you could get the team_id from the threads_in_team, right ?

I guess thread in team can give you more than one result for GM/DM, but does it matter?